nestadia/src/ppu/mod.rs

```diff
@@ -293,8 +286,9 @@ impl Ppu {
                 // VBLANK is done
                 self.status_reg.remove(registers::StatusReg::VBLANK_STARTED);
 
-                // FIXME: temporary workaround for quick and dirty, but somewhat working, rendering
-                self.dump_sprites(bus);
+                // Reset sprite 0 hit
+                self.status_reg
+                    .remove(registers::StatusReg::SPRITE_ZERO_HIT)
```

Comment:
I think it's supposed to be cleared at dot 1 of pre-render line, but maybe it doesn't change much at this place?

Reply:
I don't think it's worth the complexity of adding another branch for now until we actually try to get the timing right. Maybe I'll just add a comment for that

Reply:
Changed!